invenio_oauthclient/contrib/orcid.py

```diff
@@ -205,9 +205,14 @@ def account_setup(remote, token, resp):
     with db.session.begin_nested():
         # Retrieve ORCID from response.
         orcid = resp.get('orcid')
-
+        full_name = resp.get('name')
+        
         # Set ORCID in extra_data.
-        token.remote_account.extra_data = {'orcid': orcid}
+        token.remote_account.extra_data = {
+            'orcid': orcid,
+            'full_name': full_name
```

Comment:
So shall we extend the `user=dict()` with `user={'profile': {'full_name': resp.get('name')}` on line 173?